src/resolvers/mutation.ts

```diff
@@ -402,21 +441,13 @@ export async function replaceSavedItemTags(
   args: { input: SavedItemTagsInput[] },
   context: IContext
 ): Promise<SavedItem[]> {
+  const savedItemTagsMap = getSavedItemTagsMap(args.input);
+  const tagCreateInputs = convertToTagCreateInputs(savedItemTagsMap);
```

Comment:
aggregate/deduplicate first before even sending it to resolver. 
note: this is handled in insertTags but safe for resolver to take care of this logic.